sourcegraph/sourcegraph
# Hide Git blame information when browsing packages

Fixes https://github.com/sourcegraph/sourcegraph/issues/37768

Previously, users could toggle git blame metadata when browsing packages. However, the git blame metadata was meaningless because the git history is auto-generated for packages (every package has a single
    commit with a hardcoded author + timestamp). This commit disables
the git blame metadata toggle when browsing sources of packages.


## Test plan

Manually by starting a local server with
```
SOURCEGRAPHDOTCOM_MODE=true SOURCEGRAPH_API_URL=https://sourcegraph.com sg start web-standalone
```

* Visit package and verify that the git blame toggle is disabled  https://sourcegraph.test:3443/maven/com.squareup.okhttp3/okhttp@9ed065a545628826c82e97b1e8bc7f22d2f8b0f0/-/blob/okhttp3/HttpUrl.kt
* Visit non-package and verify that the git blame toggle is enabled, like usual. https://sourcegraph.test:3443/github.com/sourcegraph/sourcegraph/-/blob/client/web/src/components/fuzzyFinder/FuzzyFinder.tsx
* Turn on the git blame toggle in the non-package blob.
* Reload the package blob and verify that the git blame is still not displayed.
